streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBTimestampedStore.java

```diff
@@ -218,6 +219,19 @@ public void prepareBatch(final List<KeyValue<Bytes, byte[]>> entries,
             return new RocksDBDualCFIterator(name, innerIterWithTimestamp, innerIterNoTimestamp, forward);
         }
 
+        @Override
+        public KeyValueIterator<Bytes, byte[]> prefixScan(final Bytes prefix) {
+            final Bytes to = Bytes.increment(prefix);
+            return new RocksDBDualCFRangeIterator(
+                name,
+                db.newIterator(newColumnFamily),
+                db.newIterator(oldColumnFamily),
+                prefix,
+                to,
+        true,
+      false);
```

Comment:
I think here IntelliJ's formatting confused you. 🙂 
```suggestion
                to,
                true,
                false
            );
```